backend/api/brand.py

```diff
@@ -0,0 +1,135 @@
+from fastapi import APIRouter, Depends, status, Request
+
+from core.utils.middlewares import authenticate_user
+from core.schema.brand import BrandProfile
+from core.utils.errors import handle_exception, not_found_error
+
+router = APIRouter(
+    prefix="/profile",
+    tags=["brand"],
+)
+
+@router.get("/get-brand", dependencies=[Depends(authenticate_user)], status_code=status.HTTP_200_OK)
+async def get_brand(request: Request):
+    """
+    Get the brand details
+
+    Returns:
+    - dict: the response message and the brand details
+    """
+    try:
+        return {
+            "message":"Brand fetched successfully!",
+            "data": {
+                "brand_name": request.state.brand["brand_name"],
+                "brand_desc": request.state.brand["brand_desc"],
+                "brand_email": request.state.brand["brand_email"],
+                "image_url": request.state.brand["image_url"]
+            }
+        }
+    except Exception as e:
+        raise handle_exception(e)
+
+
+
+    """
+    Updates the brand description.
+
+    Parameters:
+    - request (Request): the request object
+    - brand (BrandName): Name of the brand
+
+    Returns:
+    - dict: the response message
+
+    """
+    try:
+        brand_collection = request.app.brands_collection
+        brand_email = request.state.brand["brand_email"]
+
+        if brand_collection.find_one({"brand_email": brand_email}):
+            brand_collection.update_one({"brand_email": brand_email}, {"$set": {"brand_name": brand.brand_name}})
+            return {
+                "message":"Name updated successfully!"
+            }
+        else:
+            raise not_found_error("brand")
+        
+    except Exception as e:
+        raise handle_exception(e)
+       
+@router.get("/update-image", dependencies=[Depends(authenticate_user)], status_code=status.HTTP_200_OK)
+async def update_image(request: Request):
```

Comment:
This API, and the following API for delete, would not be used in any instance, since the adding of an image, or deletion/update will perform the operation mentioned here.

It's best to remove these 2 API endpoints from here.